src/client/output/cli.rs

```diff
@@ -0,0 +1,577 @@
+        let manager_info = configuration.get_manager_info();
+        let rows = vec![
+            vec!["Worker ID".cell().bold(true), worker_id.cell()],
+            vec!["Hostname".cell().bold(true), configuration.hostname.cell()],
+            vec![
+                "Data provider".cell().bold(true),
+                configuration.listen_address.cell(),
+            ],
+            vec![
+                "Working directory".cell().bold(true),
+                configuration.work_dir.display().cell(),
+            ],
+            vec![
+                "Logging directory".cell().bold(true),
+                configuration.log_dir.display().cell(),
+            ],
+            vec![
+                "Heartbeat".cell().bold(true),
+                format_duration(configuration.heartbeat_interval)
+                    .to_string()
+                    .cell(),
+            ],
+            vec![
+                "Idle timeout".cell().bold(true),
+                configuration
+                    .idle_timeout
+                    .map(|x| format_duration(x).to_string())
+                    .unwrap_or_else(|| "None".to_string())
+                    .cell(),
+            ],
+            vec![
+                "Resources".cell().bold(true),
+                configuration.resources.summary().cell(),
+            ],
+            vec![
+                "Manager".cell().bold(true),
+                manager_info
+                    .as_ref()
+                    .map(|info| info.manager.to_string())
+                    .unwrap_or_else(|| "None".to_string())
+                    .cell(),
+            ],
+            vec![
+                "Manager Job Id".cell().bold(true),
+                manager_info
+                    .as_ref()
+                    .map(|info| info.job_id.as_str())
+                    .unwrap_or("N/A")
+                    .cell(),
+            ],
+        ];
+        let table = rows.table().color_choice(self.color_policy);
+        assert!(print_stdout(table).is_ok());
+    }
+
+    fn print_server_record(&self, server_dir: &Path, record: &AccessRecord) {
+        let rows = vec![
+            vec![
+                "Server directory".cell().bold(true),
+                server_dir.display().cell(),
+            ],
+            vec!["Host".cell().bold(true), record.host().cell()],
+            vec!["Pid".cell().bold(true), record.pid().cell()],
+            vec!["HQ port".cell().bold(true), record.server_port().cell()],
+            vec![
+                "Workers port".cell().bold(true),
+                record.worker_port().cell(),
+            ],
+            vec![
+                "Start date".cell().bold(true),
+                record.start_date().format("%F %T %Z").cell(),
+            ],
+            vec!["Version".cell().bold(true), record.version().cell()],
+        ];
+        let table = rows.table().color_choice(self.color_policy);
+        assert!(print_stdout(table).is_ok());
```

Comment:
These two lines are repeated in almost every method, let's extract them into a function.